src/publisher.ts

```diff
@@ -247,7 +262,7 @@ export class Publisher {
       topic: this.topic.name,
       messages,
     };
-    this.topic.request(
+    this.topic.request<google.pubsub.v1.PublishResponse>(
```

Comment:
This is kind of a general note about using the generated types. I'm pretty sure we are only ever going to want to use the interface versions of these. The generated callbacks and classes all reference gRPC reflection objects, which gax and grpc do not provide for us by default. There aren't very many differences between the class/interface versions, so this *might* fly under the radar, but if the interface version is available to us then I think we should use that instead.

/cc @JustinBeckwith @stephenplusplus @AVaksman